pkgs/development/python-modules/selenium/default.nix

```diff
@@ -35,8 +35,7 @@ buildPythonPackage rec {
     trio
     trio-websocket
     urllib3
-  ] ++ urllib3.optional-dependencies.secure
```

Comment:
`selenium` wants [`secure` and `socks`](https://github.com/SeleniumHQ/selenium/blob/trunk/py/setup.py#L67). Would help in the future if it's commented why `urllib3.optional-dependencies.secure` should not be added.

Reply:
I think it's a wrong requirement, selenium does not use pyopenssl (https://github.com/SeleniumHQ/selenium/search?q=pyopenssl)

According to https://urllib3.readthedocs.io/en/latest/reference/contrib/pyopenssl.html, if pyopenssl is used, there should be some code like:

```
try:
    import urllib3.contrib.pyopenssl
    urllib3.contrib.pyopenssl.inject_into_urllib3()
except ImportError:
    pass
```

Reply:
It sill has a requirement in the requirements.txt and when you package things you don't double check everything https://github.com/SeleniumHQ/selenium/blob/trunk/py/requirements.txt#L20